hawk/app/assets/stylesheets/shared/_misc.scss

```diff
@@ -32,3 +32,30 @@
     border-radius: 20px;
   }
 }
+
+.diff_sub {
+  background: #fdd;
+}
+
+.diff_add {
+  background: #dfd;
+}
+
+.diff_chg {
+  background: #ddf;
+}
+
+
+td.diff_header {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `diff_header` should be written in lowercase with hyphens